corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py

```diff
@@ -7,55 +7,109 @@
 from corehq.apps.app_manager.dbaccessors import get_app_ids_in_domain
 from corehq.apps.app_manager.models import Application, PreloadAction
 from corehq.apps.app_manager.util import save_xform
-from corehq.apps.app_manager.xform import XForm
+from corehq.apps.app_manager.xform import XForm, SESSION_USERCASE_ID
+from corehq.toggles import NAMESPACE_DOMAIN, USER_PROPERTY_EASY_REFS
 
 
 logger = logging.getLogger('app_migration')
 logger.setLevel('DEBUG')
 
 
 class Command(BaseCommand):
-    args = 'app_id_or_domain'
     help = '''
         Migrate apps from case management in the app builder to form builder.
-        Pass either a domain name (to migrate all apps in the domain) or an
-        individual app id. Will skip any apps that have already been migrated.
+        Pass either domain name(s) (to migrate all apps in the domain) or
+        individual app id(s). Will skip any apps that have already been
+        migrated.
     '''
 
+    def add_arguments(self, parser):
+        parser.add_argument('app_id_or_domain', nargs='+',
+            help="App ID or domain name. Must be a domain name "
+                 "with --usercase option.")
+        parser.add_argument('--usercase', action='store_true',
+            help='Migrate user properties.')
+
     def handle(self, *args, **options):
+        domains = []
         app_ids = []
-        try:
-            Application.get(args[0])
-            app_ids = [args[0]]
-        except ResourceNotFound:
-            app_ids = get_app_ids_in_domain(args[0])
-            logger.info('migrating {} apps in domain {}'.format(len(app_ids), args[0]))
+        self.migrate_usercase = options["usercase"]
+        for ident in options["app_id_or_domain"]:
+            if not self.migrate_usercase:
+                try:
+                    Application.get(ident)
+                    app_ids.append(ident)
+                    continue
+                except ResourceNotFound:
+                    pass
+            ids = get_app_ids_in_domain(ident)
+            app_ids.extend(ids)
+            if ids:
+                domains.append(ident)
+            logger.info('migrating {} apps in domain {}'.format(len(ids), ident))
 
         for app_id in app_ids:
             logger.info('migrating app {}'.format(app_id))
             self.migrate_app(app_id)
 
+        if self.migrate_usercase:
+            for domain in domains:
+                USER_PROPERTY_EASY_REFS.set(domain, True, NAMESPACE_DOMAIN)
```

Comment:
Do you mean while the management command is running? Like after it converts the user properties for all the apps in the domain, but before this line is executed? I assumed the process would be fast enough that it wouldn't be a big deal, but maybe I'm wrong. Also, possibly more concerning is if there was an error migrating an app (that is not the first app to be migrated in the domain). In that case some apps would be migrated and others would not be, and the flag would not be set because the command errored out. I could see that being a problem, but I'm not sure how to work around it. Maybe wait to save the apps until they've all been migrated? -- although even then we run into problems because we don't have a real transactional database.